haystack/pipeline.py

```diff
@@ -1287,4 +1288,31 @@ def run(self, query, documents, **kwargs):
         # Pass also the other incoming kwargs so that future nodes still have access to it
         output.update(**kwargs)
 
-        return output, "output_1"
\ No newline at end of file
+        return output, "output_1"
+
+
+class MostSimilarDocumentsPipeline(BaseStandardPipeline):
+    def __init__(self, document_store: BaseDocumentStore):
+        """
+        Initialize a Pipeline for finding similar documents using document ids.
+
+        :param document_store: Document Store instance
+        """
+        self.document_store = document_store
+
+    def run(self, document_ids: List[str], top_k: int = 5):
+        """
+        :param document_ids: document ids
+        :param top_k: How many documents id to return against single document
+        """
+        return_ids: list = []
+        self.document_store.return_embedding = True  # type: ignore
```

Comment:
Let's set `return_embedding` back to false at the end of this method to prevent any side effects with other methods that don't expect the documentstore to return embeddings